docs/configuration-reference/components/istio-operator.md

```diff
@@ -0,0 +1,60 @@
+# Istio operator configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+Istio is an open-source cloud native service mesh. It uses sidecar pattern to inject Envoy proxies in the pods, which acts as data plane of the Istio service mesh. Control plane components take in user input and implement policies using Envoy proxy.
+
+This component installs the Istio operator.
+
+> **NOTE**: This is an **unsupported and experimental component**. It is not recommended to use this in production. The UX of this component can change anytime as it is tested overtime. If there are fundamental problems with the component then it could be removed without prior notice.
+
+## Prerequisites
+
+* A Lokomotive cluster accessible via `kubectl`.
+
+## Configuration
+
+Istio operator component configuration example:
+
+```tf
+component "experimental-istio-operator" {
+  enable_monitoring = true
+  profile           = "minimal"
+}
+```
+
+## Attribute reference
+
+Table of all the arguments accepted by the component.
+
+Example:
```

Comment:
What does this Example mean here?

Reply:
Good question. I am not sure what they mean. It is a remnant of the copy-paste of the skeleton from other components.